### ISSUE TITLE
[preset] Create a preset for release

### ISSUE THREAD

**Issue by mhs4670go** [Samsung/ONE#10553]
@mhs4670go , FYI, if preset settings are changed after release, inconvenience will occur in reproducing the corresponding release. It seems that this part also needs to be considered in the release process. 

Alternatively, it is also a way to make the operation of the preset settings different. That is, presets are never modified, so that new presets are always added when necessary.

_Originally posted by @lemmaa in https://github.com/Samsung/ONE/issues/10541#issuecomment-1484402470_

---

It would be better to have a preset for release.

We can make a preset named "r1.22.0", "r1.23.0", etc.

```bash
$ ./nnas create-package --preset r1.22.0
```

**Comment by mhs4670go** [comment]
@seanshpark @jinevening What is your take on this?

**Comment by lemmaa** [comment]
> It would be better to have a preset for release.

Personally, as suggested at the end, I think it's good to create a new one without modifying the existing one when changes are needed to the preset.

One concern is how burdensome it is to add a new preset. Wouldn't it be less burdensome than creating a preset for every release?

**Comment by seanshpark** [comment]
Initial design (not by me) was to make new `preset` for every release; which was a burden.
And there was some changes that was a bit hard to follow, lazy me...

> inconvenience will occur in reproducing the corresponding release. 

this is the pain point to reproduce release just before the change of `preset`.
I thought to checkout to the version before the change.
Can you do this? or what may be the problem of doing so?

**Comment by jinevening** [comment]
I've found the number of commits applied to recent five presets

`git log --oneline XX | wc -l` (XX is the preset file)

20221125 (4 commits)
20220323 (12 commits)
20210910 (1 commit)
20210706 (1 commit)
20210406 (3 commits)

20220323 has quite many changes. But other presets were updated not that frequently.

**Comment by seanshpark** [comment]
looks valid but not sure what to do
